### DISCUSSION TITLE
FLIF support?

### DISCUSSION THREAD

**Discussion by unquietwiki** [d2phap/ImageGlass#282] [General] [answered]
- http://flif.info/
- https://github.com/FLIF-hub/FLIF

I stumbled across this image format tonight. It looks like a fairly decent replacement for JPG, PNG, and WEBP. There's at least one C# library for it, but it currently uses an older version of the code-base.

**Comment by d2phap** [comment, marked as answer]
Hi @unquietwiki 

I found a WIC codec: https://github.com/peirick/FlifWICCodec
Because ImageGlass 4.5 is using WIC, so you can install this codec to view FLIF file. 
Then, you can add new extension in ImageGlass by opening Settings > File Associations > Add
